crates/router/src/core/metrics.rs

```diff
@@ -44,3 +44,23 @@ counter_metric!(
     WEBHOOK_EVENT_TYPE_IDENTIFICATION_FAILURE_COUNT,
     GLOBAL_METER
 );
+
+counter_metric!(ROUTING_CREATE_REQUEST_RECEIVED, GLOBAL_METER);
+counter_metric!(ROUTING_CREATE_RESPONSE, GLOBAL_METER);
+counter_metric!(ROUTING_MERCHANT_DICTIONARY_RETRIEVE, GLOBAL_METER);
+counter_metric!(ROUTING_MERCHANT_DICTIONARY_RETRIEVE_RESPONSE, GLOBAL_METER);
```

Comment:
Let's make it a little more specific, rather than doing something like
`ROUTING_MERCHANT_DICTIONARY_RETRIEVE_RESPONSE`
make it as
`ROUTING_MERCHANT_DICTIONARY_RETRIEVE_SUCCESS_RESPONSE` 

Let's do this for all the `counter_metric!` we have created.

Reply:
Have updated